### PR TITLE
Added test for getting an error message from RestJsonError

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         "matthiasnoback/symfony-config-test": "^4.1",
         "nette/php-generator": "^3.3",
         "nette/utils": "^3.0",
-        "nyholm/symfony-bundle-test": "^1.6",
+        "nyholm/symfony-bundle-test": "^1.6.1",
```
Comment (Nyholm) [inline composer.json]: This will make sure dep-lowest does not fail

File: src/Core/Test/Http/SimpleMockedResponse.php [hunk 2]
```diff
@@ -48,14 +51,29 @@ public function cancel(): void
 
     public function getInfo(string $type = null)
     {
-        if (null === $type || 'response_headers' === $type) {
-            return [];
+        $info = [
+            'response_headers' => $this->getFlatHeaders(),
```
Comment (jderusse) [inline src/Core/Test/Http/SimpleMockedResponse.php]: if info is `http_code`, we compute that for nothing :-/

Comment (Nyholm) [reply src/Core/Test/Http/SimpleMockedResponse.php]: It is just tests ¯\_(ツ)_/¯

Comment (Nyholm) [reply src/Core/Test/Http/SimpleMockedResponse.php]: I also expect it to be more things to this array later.

**Comment by Nyholm** [comment]
FYI: Im doing all these updates because "Lowest dependencies" should not fail anymore.

**Pull request merged**